Kitodo-DataManagement/src/main/java/org/kitodo/config/ConfigMain.java

```diff
@@ -42,16 +46,23 @@ private static PropertiesConfiguration getConfig() {
             synchronized (ConfigMain.class) {
                 PropertiesConfiguration initialized = config;
                 if (initialized == null) {
-                    PropertiesConfiguration.setDefaultListDelimiter('&');
                     try {
-                        initialized = new PropertiesConfiguration(CONFIG_FILE);
+                        // Create and initialize the builder
+                        ReloadingFileBasedConfigurationBuilder<PropertiesConfiguration> builder =
+                            new ReloadingFileBasedConfigurationBuilder<>(PropertiesConfiguration.class)
+                                .configure(new Parameters().properties()
+                                    .setFileName(CONFIG_FILE)
+                                    .setListDelimiterHandler(new DefaultListDelimiterHandler('&'))
+                                    .setIncludesAllowed(false));
+                        // Register an event listener for triggering reloading checks
+                        builder.addEventListener(ConfigurationBuilderEvent.CONFIGURATION_REQUEST,
+                            (EventListener) event -> builder.getReloadingController().checkForReloading(null));
```

Comment:
```suggestion
                            event -> builder.getReloadingController().checkForReloading(null));
```
Is this cast necessary? See above.